media-fonts/fontawesome/fontawesome-5.13.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN="Font-Awesome"
+inherit font
+
+DESCRIPTION="The iconic font"
+HOMEPAGE="https://fontawesome.com"
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/FortAwesome/${MY_PN}.git"
+	KEYWORDS=""
```

Comment:
Please drop the KEYWORDS from this part altogether. It messes up with ekeyword tool.

Reply:
Dropped :)